pkg/lwip/contrib/sock/udp/lwip_sock_udp.c

```diff
@@ -113,12 +130,10 @@ ssize_t sock_udp_recv(sock_udp_t *sock, void *data, size_t max_len,
         memcpy(&remote->addr, &buf->addr, addr_len);
         remote->port = buf->port;
     }
-    /* copy data */
-    for (struct pbuf *q = buf->p; q != NULL; q = q->next) {
-        memcpy(data_ptr, q->payload, q->len);
-        data_ptr += q->len;
-    }
-    netbuf_delete(buf);
+    assert(buf->p->next == NULL);
+    assert(buf->p->len == res);
```

Comment:
Sadly, this doesn't work :-/ as soon as the received UDP payload is larger then 530 with IPv6, this assertions are hit :-(. I'll try to find out if there is another way out.

Reply:
Is this still the case?

Reply:
Nope. This was what #13779 and the latest fixups were about.